ports/raspberrypi/boards/maple_elite_pi/mpconfigboard.mk

```diff
@@ -0,0 +1,9 @@
+USB_VID = 0xFFFF
```

Comment:
A proper VID/PID combination must be provided for this board.

Reply:
Yup, that's an open question. I'm not convinced that Maple Computing or Keebio bothered to get a VID/PID.

I'm going to check a few different sources and see what the common use is.

Reply:
The UF2 bootloader comes up as 2e8a:0003.

The other major use is the QMK keyboard firmware, which defines VID/PID by the keyboard, not the controller.

The only contact I can find for Maple Computing themselves is a reddit user (probably). Otherwise, it looks like Keebio (@nooges) has been doing most of the work (eg, contributing support to QMK).

I would appreciate advice. I know of https://pid.codes/ but the Elite-Pi is not an open hardware project.

Reply:
It looks like Maple Computing is @That-Canadian

Reply:
Yes that's me =) great to see this picked up here! For context, I mainly do hardware design, and Keebio (@nooges) generally has been helping by adding files to QMK / software-related-things.

As far as VID/PID goes, no we did not formally get a VID/PID heh, it's mainly being used in QMK, which as you pointed out above defined VID/PID by keyboard not the specific controller.

Let me know what I can do here to assist.

Reply:
Quick follow-up, looks like the Raspberry Pi foundation will allow people to acquire a PID using their VID : https://github.com/raspberrypi/usb-pid

I'll look into this further and pursue it if needed over the next few days.

Reply:
Yes, a VID/PID is needed for this to move forward.

I noticed in other PRs that CircuitPython prefers that the VID/PID pair should be unique to CircuitPython if the device has multiple operating modes.

Reading their docs, the Raspberry Pi Foundation might want to issue a generic PID for RP2040 boards running CircuitPython? A possibility to discuss with them.

Reply:
Its ok if the PID isn't for CP only. We would like the PID to be unique within CP though.